content/overview/security/for-assessors.md

```diff
@@ -0,0 +1,59 @@
+---
+menu:
+  overview:
+    parent: technology
+title: Overview for assessors
+weight: 50
+aliases:
+  - /docs/intro/security/for-assessors
+  - /intro/security/for-assessors
+  - /overview/security/for-assessors/
+---
+
+## How cloud.gov works
+
+cloud.gov is a platform as a service. This means customers provide the code for their app and we provide the infrastructure, operating system, and supporting technologies required to run the app. There are three basic components to any cloud.gov app:
+
+1. Hardware infrastructure provided by Amazon Web Services's GovCloud environment including web servers, relational database servers, DNS routing services, and other systems the customer might need.
```

Comment:
AWS provides both hardware and software infrastructure - lots of software going on there, just infrastructure-level software instead of platform-level software.